html/Mogo/js/sections-animation/helper-functions.js

```diff
@@ -0,0 +1,45 @@
+import {components} from './menu-elements.js';
+
+const {navbar, navbarMenu} = components;
+
+function setNavbarBackground() {
+  navbar.classList.add('navbar--with-bg');
+}
+
+function removeNavbarBackground() {
+  navbar.classList.remove('navbar--with-bg');
+}
+
+function changeNavbarBackground() {
+  if (window.scrollY >= 200) {
+    setNavbarBackground();
+  } else if (!navbarMenu.classList.contains('navbar__menu--active')) {
+    removeNavbarBackground();
+  }
+}
+
+function addScrollEffect(event) {
+  const elementName = event.getAttribute('href');
+  const element = document.querySelector('#' + elementName);
+  window.scrollTo({
+    behavior: 'smooth',
+    left: 0,
+    top: element.offsetTop,
+  });
+}
+
+function addEventToMenuItems() {
+  Array.from(navbarMenu.getElementsByTagName('li')).forEach((item) => {
```

Comment:
it is possible to have just one listener using [event delegation](https://javascript.info/event-delegation), which is generally very useful technic, which based on [bubbling and capturing of events](https://javascript.info/bubbling-and-capturing)